apps/ensadmin/src/components/ensnode/types.ts

```diff
@@ -12,7 +13,12 @@ export namespace EnsNode {
   /**
    * The status of the ENS node.
    */
-  export interface Metadata extends Omit<PonderMetadata.MetadataMiddlewareResponse, "env"> {
+  export interface Metadata extends PonderMetadata.MetadataMiddlewareResponse {
+    /** Application dependencies info */
+    deps: PonderMetadata.MetadataMiddlewareResponse["deps"] & {
+      ensRainbow: EnsRainbow.VersionInfo;
+    };
```

Comment:
We declare application-level type to specify a generic type coming from the `ponder-metadata` package.